crates/base/src/deno_runtime.rs

```diff
@@ -170,8 +173,16 @@ impl DenoRuntime {
             let arc_emitter_factory = Arc::new(emitter_factory);
 
             let main_module_url_file_path = main_module_url.clone().to_file_path().unwrap();
+
+            let maybe_code = if only_module_code {
```

Comment:
can't we directly pass `maybe_module_code` as an option to `generate_binrary_eszip`

Reply:
It's under `only_module_code` so that ```        let only_module_code =
            maybe_module_code.is_some() && maybe_eszip.is_none() && !is_some_entry_point;``` it only applies for this case.